package.json

```diff
@@ -63,7 +62,9 @@
         "extension:package": "npm run extension:clean && vsce package --baseContentUrl https://raw.githubusercontent.com/atlassian/atlascode/main/ --allow-star-activation",
         "extension:install": "npm run extension:package && code --install-extension ./atlascode-*.vsix --force",
         "test": "jest",
-        "test:ui": "npm run compile:ui-test && extest run-tests './generated/ui-test/*.test.js' --code_version max --code_settings ui-test-settings.json --extensions_dir .test-extensions",
+        "e2e:compile": "tsc --project e2e/tsconfig.e2e.json",
```

Comment:
I feel that anything that is related to testing should be prefixed with the word "test"

Aka, these commands should look like this:

`test:e2e:compile`
`test:e2e`
`test:e2e:rerun`

Reply:
No problem! If that's alright with you, for convenience I'll address this in a follow-up PR to not rebase too much :D

Reply:
Addressed the command naming in the [follow-up PR](https://github.com/atlassian/atlascode/pull/36)